language/en

```diff
@@ -180,10 +180,11 @@
 			"add": "Add this Page",
 			"delete": "Delete / re-Arrange Pages",
 			"swap": "Change this Page",
-			"sort": "Drag to re-arrange or delete pages"
+			"sort": "Drag to re-arrange or delete pages";
```

Comment:
@RukayaDavids you should not use a semi-colon `;` to separate object entries, instead use a comma `,`.

In this case it's the last entry of an object in which case you do not need a comma, and adding a comma here will break the JSON format. Please remove here.